hack/ci-e2e.sh

```diff
@@ -32,58 +43,100 @@ minikube start
 # Load the BMO e2e image into it
 minikube image load quay.io/metal3-io/baremetal-operator:e2e
 
-# Start VBMC
-docker run --name vbmc --network host -d \
-  -v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
-  -v /var/run/libvirt/libvirt-sock-ro:/var/run/libvirt/libvirt-sock-ro \
-  quay.io/metal3-io/vbmc
-
 # Create libvirt domain
 VM_NAME="bmo-e2e-0"
-BOOT_MAC_ADDRESS="00:60:2f:31:81:01"
-VBMC_PORT="16230"
-virt-install --connect qemu:///system -n "${VM_NAME}" --description "Virtualized BareMetalHost" --osinfo=ubuntu-lts-latest \
-  --ram=4096 --vcpus=2 --disk size=20 --graphics=none --console pty --serial pty --pxe \
-  --network network=baremetal-e2e,mac="${BOOT_MAC_ADDRESS}" --noautoconsole
+export BOOT_MAC_ADDRESS="00:60:2f:31:81:01"
+
+virt-install \
+   --connect qemu:///system \
+   --name "${VM_NAME}" \
+   --description "Virtualized BareMetalHost" \
+   --osinfo=ubuntu-lts-latest \
+   --ram=4096 \
+   --vcpus=2 \
+   --disk size=20 \
+   --graphics=none \
+   --console pty \
+   --serial pty \
+   --pxe \
+   --network network=baremetal-e2e,mac="${BOOT_MAC_ADDRESS}" \
+   --noautoconsole
 
-# Add BMH VM to VBMC
-docker exec vbmc vbmc add "${VM_NAME}" --port "${VBMC_PORT}"
-docker exec vbmc vbmc start "${VM_NAME}"
-docker exec vbmc vbmc list
-
-# These variables are used by the tests. They override variables in the config file.
 # This IP is defined by the network we created above.
-# Together with the VBMC_PORT this becomes the BMC_ADDRESS used by the BMH in the test.
 IP_ADDRESS="192.168.222.1"
-export BMC_ADDRESS="ipmi://${IP_ADDRESS}:${VBMC_PORT}"
-export BOOT_MAC_ADDRESS
+
+# These variables are used by the tests. They override variables in the config file.
 # These are the VBMC defaults (used since we did not specify anything else for `vbmc add`).
 export BMC_USER=admin
 export BMC_PASSWORD=password
-CIRROS_VERSION="0.6.2"
-IMAGE_FILE="cirros-${CIRROS_VERSION}-x86_64-disk.img"
-export IMAGE_CHECKSUM="c8fc807773e5354afe61636071771906"
-export IMAGE_URL="http://${IP_ADDRESS}/${IMAGE_FILE}"
-IMAGE_FOLDER="${REPO_ROOT}/test/e2e/images"
 
-## Setup image server
-# Create a directory for images
+if [[ "$USE_TOOL" == "vbmc" ]]; then
+	# Start VBMC
+	docker run --name vbmc --network host -d \
+	-v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
```

Comment:
```suggestion
        	-v /var/run/libvirt/libvirt-sock:/var/run/libvirt/libvirt-sock \
```
Easier to read if you indent multi-line commands.